oaknational/Oak-Web-Application
# feat: 1436 Add event tracking

## Description

- `pageView` not implemented yet - will be done in a separate ticket
- `resourcesDownloaded` event currently comes out incorrectly - waiting on confirmation on download types 

### Testing steps:
- check that the events are coming through in Avo as `development` events
(https://www.avo.app/schemas/5PhajbVijwhXVKIJtGMT/branches/cjVJMBMUT/events)
- check that events are coming through to Posthog

## Issue(s)

Fixes #1436
